web_common/web_common/styles/typography.scss

```diff
@@ -1,32 +0,0 @@
-@import "variables/colors.scss";
```

Comment:
again, not sure why this left

Reply:
I'm confused, it's deleted.

Reply:
I just mean, why did typeography get deleted? Now the Hail logo is just body font-size.